README.md

```diff
@@ -26,6 +26,11 @@ Please check out common DSC Resources [contributing guidelines](https://github.c
 * Initial release with the following resources 
     - xPhp
 
+### 1.0.?
```

Comment:
Pester use 'unreleased' version for that. I don't think picking the name make difference. Call it vnext or unreleased or ???.